calliope/config/defaults.yaml

```diff
@@ -89,7 +89,7 @@ tech_groups:
     supply:
         required_constraints: []
         allowed_constraints: ['energy_cap_equals', 'energy_cap_equals_systemwide', 'energy_cap_max', 'energy_cap_max_systemwide', 'energy_cap_min', 'energy_cap_min_use', 'energy_cap_per_unit', 'energy_cap_scale', 'energy_eff', 'energy_prod', 'energy_ramping', 'export_cap', 'export_carrier', 'force_resource', 'lifetime', 'resource', 'resource_area_equals', 'resource_area_max', 'resource_area_min', 'resource_area_per_energy_cap', 'resource_min_use', 'resource_scale', 'resource_unit', 'units_equals', 'units_equals_systemwide', 'units_max', 'units_max_systemwide', 'units_min']
-        allowed_group_constraints: [demand_share_min, demand_share_max, demand_share_equals, demand_share_per_timestep_min, demand_share_per_timestep_max, demand_share_per_timestep_equals, demand_share_per_timestep_decision, supply_share_min, supply_share_max, supply_share_equals, supply_share_per_timestep_min, supply_share_per_timestep_max, supply_share_per_timestep_equals, cost_max, cost_min, cost_equals, cost_var_max, cost_var_min, cost_var_equals, cost_investment_min, cost_investment_max, cost_investment_equals, energy_cap_share_min, energy_cap_share_max, energy_cap_min, energy_cap_max, resource_area_min, resource_area_max, supply_min, supply_max]
+        allowed_group_constraints: [demand_share_min, demand_share_max, demand_share_equals, demand_share_per_timestep_min, demand_share_per_timestep_max, demand_share_per_timestep_equals, demand_share_per_timestep_decision, supply_share_min, supply_share_max, supply_share_equals, supply_share_per_timestep_min, supply_share_per_timestep_max, supply_share_per_timestep_equals, cost_max, cost_min, cost_equals, cost_var_max, cost_var_min, cost_var_equals, cost_investment_min, cost_investment_max, cost_investment_equals, energy_cap_share_min, energy_cap_share_max, energy_cap_share_equals, energy_cap_min, energy_cap_max, energy_cap_equals, resource_area_min, resource_area_max, resource_area_equals, carrier_prod_min, carrier_prod_max, carrier_prod_equals]
```

Comment:
We should rename all `demand_share` to `carrier_con_share` and `supply_share` to `carrier_prod_share` for consistency

Reply:
I don't think carrier_con_share makes sense, sadly, as we only consider demand, not other energy consuming techs (storage, conversion, and transmission)...